src/utils.js

```diff
@@ -107,6 +107,35 @@ exports.transformAdServerTargetingObj = function (targeting) {
   }
 };
 
+/**
+ * Read an adUnit object and return the sizes used in an [[728, 90]] format (even if they had [728, 90] defined)
+ * Preference is given to the `adUnit.mediaTypes.banner.sizes` object over the `adUnit.sizes`
+ * @param {object} adUnit one adUnit object from the normal list of adUnits
+ * @returns {array[array[number]]} array of arrays containing numeric sizes
+ */
+export function getAdUnitSizes(adUnit) {
```

Comment:
any reason this is a new function vs reusing `parseSizesInput` ?

Reply:
that function converts the sizes into an array of strings like `["300x250"]`.  I just wanted the actual sizes to reassign into the register call and not have to parse it again.